examples/otlp-push-controller/README.md

```diff
@@ -0,0 +1,30 @@
+# Autometrics + OTLP push controller
+
+This example demonstrates how you can push autometrics via OTLP gRPC protocol to an OTEL-Collector or another compatible solution.
+
+## ⚠️ Warning
```

Comment:
I'm not sure this is the best place to put this warning because it's not a warning related to this example, but more a warning about if you're going to copy some of this code into your own project. It might be better to put it as comment in the `Cargo.toml` file and in the source code